docs/src/main/paradox/bootstrap/index.md

```diff
@@ -186,7 +186,7 @@ and the operation will (presumably) eventually succeed. You'll want to specify t
 #### Graceful shutdown 
```

Comment:
> I'm a bit in doubt if this should be a new main section on the docs or a sub-section under the bootstrap one.

I think it would be good with a separate Rolling Update page, and move some sections from here to that page. We can have link from the bootstrap page. The new feature "Kubernetes Rolling Update" can be part of that page.